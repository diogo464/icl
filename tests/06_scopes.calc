{
    let x = 10;
    {
        let x = 20;
        {
            let x = 30;
            println x;
            x := 40;
            println x;
        };
        println x;
        x := 50;
        println x;
    };
    println x;
    x := 60;
    println x;
};;