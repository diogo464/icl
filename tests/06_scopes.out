30.0
40.0
20.0
50.0
10.0
60.0
